Lib/code.py

```diff
@@ -276,8 +315,20 @@ def raw_input(self, prompt=""):
         return input(prompt)
 
 
+class Quitter:
+    def __init__(self, name):
+        self.name = name
+        self.eof = 'Ctrl-Z plus Return' if sys.platform == "win32" else \
+                   'Ctrl-D (i.e. EOF)'
```

Comment:
Nit: I personally prefer an if-else to a ternary with a backslash. Also, `site.py` uses `os.pathsep` for this purpose... so let's maybe just be consistent with that?

```suggestion
        if os.sep == '\\':
            self.eof = "Ctrl-Z plus Return"
        else:
            self.eof = "Ctrl-D (i.e. EOF)"
```

Reply:
I'm okay with the if/else block instead of an inline statement, but I kind of changed the OS check on purpose.

The fundamental thing we want to know here is what's the EOF on a specific system, and the best effort we can make is to get the OS type - if it's Windows then it's Ctrl+Z + Return. `os.sep` is a design decision(finger print?) for Windows, the logic would be: `os.sep == '\\'` -> "this is Windows!". The seperator itself has nothing to do with what the eof directly, it just happens to be linked together because of Windows.

For similar checks, there are a lot of usage of ``sys.platform == "win32"`` in the standard library, and almost only `site.py` uses `os.sep` to figure out the OS platform. So I think `sys.platform` is actually preferred in this case.

Reply:
Could the site.py code be a holdover from DOS, which isn’t Windows but still uses \ ? Presumably other antique OS’es using \ also used ^Z ? Not that it matters today. :-)

Reply:
Too bad I'm not young enough to ask the question "what is DOS". That's a possibility, but the connection between `\\` and Ctrl+Z is still indirect. I don't know if we can build modern Python on ancient OS anymore, always assume that Windows is the only outlier.